dockerizeit/master/jenkins.properties

```diff
@@ -14,6 +14,13 @@ credentials {
     description = ""
     path = "/var/jenkins_home/.ssh/.password"
   }
+  smtp {
```

Comment:
Yes, I agree. Didn't think about it that way. I will correct it and push tomorrow to the branch (as you mentioned in comment below)